### PR TITLE
docs: Write a Contributing guide

### PR DESCRIPTION
**Pull request by wakamsha**
## Summary

Created a documentation to explain how anyone that considering contributing to this project can do so.

The contents of the document are written with reference to [the description in `facebook/stylex`](https://github.com/facebook/stylex/blob/main/.github/CONTRIBUTING.md) and the support of ChatGPT.

## References

- https://github.com/moneyforward/frontend-forward/issues/77

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Contributing
+
+I appreciate your considering contributing `eslint-config-moneyforward`. This document is a guide to help make your contribution easier.
+
+## Getting Started
+
+### Prerequisites
+
+- [Node.js](https://nodejs.org) (See [`.node-version`](/moneyforward/eslint-config-moneyforward/.node-version))
```
Comment (taigakiyokawa) [inline .github/CONTRIBUTING.md:9]: The link for `.node-version` seems wrong. I moved to https://github.com/moneyforward/eslint-config-moneyforward/blob/docs/contribute-guide/moneyforward/eslint-config-moneyforward/.node-version on click the link.

image

Fix the link like my suggestion or replace it with the completed URL instead.

```suggestion
- [Node.js](https://nodejs.org) (See [`.node-version`](/.node-version))
```

Comment (taigakiyokawa) [reply .github/CONTRIBUTING.md:9]: Some other links to this repository (specified like `moneyforward/eslint-config-moneyforward/*`) also seem to jump to the wrong pages.

**Review by taigakiyokawa — CHANGES_REQUESTED** [review]
Please check about internal links behavior.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (wakamsha) [reply .github/CONTRIBUTING.md]: fixed: [436e0ce](https://github.com/moneyforward/eslint-config-moneyforward/pull/304/commits/436e0ce4622ba16cccf15f7f2723d0fa3bcb049e)

**Review by taigakiyokawa — APPROVED** [review]
LGTM 🙆‍♂️

**Pull request merged**